scripts/bootstrap-linux.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+#
+# Boot2Gecko Linux bootstrap script
+# See --help
+
+this_dir=$(cd `dirname $0`; pwd)
+
+print_usage() {
+    cat << EOF
+Boot2Gecko Linux Bootstrap
+
+This script attempts to bootstrap a "minimal" Linux installation
+with the tools necessary to build Boot2Gecko.
+
+Usage: $0 [options]
+Options:
+    --help              print this message
+    --dry-run           only prints commands instead of executing them
+                        (default: run commands)
+    --auto-install      automatically installs all necessary packages,
+                        and doesn't prompt (default: prompt)
+    --clone             clone the Boot2Gecko git repository after the
+                        environment has been bootstrapped into:
+                        $PWD/B2G
+EOF
+    exit 1
+}
+
+parse_options() {
+    option_dry_run=no
+    option_auto_install=no
+    option_help=no
+    option_clone=no
+
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            --help|-h) option_help=yes;;
+            --dry-run) option_dry_run=yes;;
+            --auto-install) option_auto_install=yes;;
+            --clone) option_clone=yes;;
+	    --check) option_check=yes;;
+            *) break;;
+        esac
+        shift
+    done
+
+    if [ "$option_help" = "yes" ]; then
+        print_usage
+    fi
+}
+
+run_command() {
+    if [ "$option_dry_run" = "yes" ]; then
+        command_prefix="(dry-run) "
+    fi
+
+    echo "$command_prefix=> $@"
+
+    if [ "$option_dry_run" = "no" ]; then
+        $@
+    fi
+}
+
+clone_b2g() {
+    echo ""
+    echo "Clone selected"
+    echo ""
+    if [ -d "$PWD/B2G" ]; then
+        echo "Found existing B2G: $PWD/B2G"
+        return 0
+    fi
+    run_command git clone git://github.com/mozilla-b2g/B2G.git
+}
+
+auto_install() {
+	echo ""
+	echo "Auto install selected"
+	echo ""
+	# Validate the host OS and perform platform specific validation.
+	os=$(grep -R 'NAME=' /etc/os-release | head -1 | cut -d '"' -f 2)
```

Comment:
On Fedora, this results in `os="NAME=Fedora"`...

`grep '^NAME=' /etc/os-release | sed "s/NAME=//"` is similar but I know that it works on Fedora